packages/a11y-testing/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@fluentui/a11y-testing",
+  "version": "0.1.0",
+  "description": "Fluent UI tests for a11y patterns.",
+  "main": "lib-commonjs/index.js",
```

Comment:
From what I can tell, this package should have used the `node` template not the `react` template since it's intended to be used for testing only, not run in the browser (though I agree the template name is a little misleading). So now there's a bunch of configuration that's not quite right. 

Is it okay if I push an update to your PR fixing the config issues? (would be easier than trying to explain them all)

Reply:
@ecraig12345 
please feel free to push update to this PR :) 
I appreciate you help. :) as I have no experience with creating the package. 
Even if you have any documentation where I could read more about creating package I would be happy to read it.

Reply:
It's not documented 😬 but that would be a good idea since we're adding more packages now! I'm also planning to update the create-package script with more options (and clearer explanations) so it's easy for people to get the right setup in more cases.